geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/formats/FeatureExporter.scala

```diff
@@ -8,10 +8,10 @@
 
 package org.locationtech.geomesa.tools.export.formats
 
-import java.io.{Closeable, Flushable}
+import java.io.Closeable
 
 import org.geotools.data.simple.SimpleFeatureCollection
 
-trait FeatureExporter extends Closeable with Flushable {
+trait FeatureExporter extends Closeable {
```

Comment:
it wasn't actually used, it just flushed at the end, but having to allow the ability to flush complicated things.